src/commands/org/create/shape.ts

```diff
@@ -12,7 +12,7 @@ import {
   loglevel,
 } from '@salesforce/sf-plugins-core';
 import { Messages, Connection, Logger } from '@salesforce/core';
-import { SaveResult } from 'jsforce';
+import type { SaveResult } from 'jsforce';
```

Comment:
fancy

Reply:
it all should disappear in the js, but this helps some tooling (ex :import cost analysis)